src/core/server/server.api.md

```diff
@@ -401,31 +404,22 @@ export interface LogRecord {
 // Warning: (ae-forgotten-export) The symbol "OnPostAuthResult" needs to be exported by the entry point index.d.ts

```

Comment:
Shouldn't we export this?

Reply:
`OnPostAuthResult`, `OnPreAuthResult`, `RouterRoute`
they are internal core types, plugins don't have access to them.